swap/src/bin/asb.rs

```diff
@@ -107,6 +107,22 @@ async fn main() -> Result<()> {
             )
             .unwrap();
 
+            tokio::spawn(async move {
+                while let Some(swap) = swap_receiver.recv().await {
+                    tokio::spawn(async move {
+                        let swap_id = swap.swap_id;
+                        match run(swap).await {
+                            Ok(state) => {
+                                tracing::debug!(%swap_id, "Swap finished with state {}", state)
+                            }
+                            Err(e) => {
+                                tracing::error!(%swap_id, "{:#}", e)
```

Comment:
```suggestion
                                tracing::error!(%swap_id, "Swap failed with {:#}", e)
```

Reply:
Note: The `with_context` inside the run function caused problems with the lifetime of the database, that's why I remove it. Better to be added here anyway.